route_info_zos.go

```diff
@@ -0,0 +1,46 @@
+// +build zos
+
+package sockaddr
+
+import (
+   "errors"
+   "os/exec"
+   "regexp"
+   "strings"
+)
+
+var defaultRouteRE *regexp.Regexp = regexp.MustCompile(`^Default +([0-9\.\:]+) +([^ ]+) +([0-9]+) +([^ ]+)`)
+
+func NewRouteInfo() (routeInfo, error) {
+	return routeInfo{}, nil
+}
+
+// zosGetDefaultInterfaceName executes the onetstat command and returns its output
+func zosGetDefaultInterfaceName() (string, error) {
+	out, err := exec.Command("/bin/onetstat", "-r").Output()
+	if err != nil {
+		return "", err
+	}
+	return string(out), nil
+}
+
+// zosProcessOneStatOutput processes the output of onetstat -r and returns the default interface name
+func zosProcessOnetstatOutput(output string) (string, error) {
+	linesout := strings.Split(output, "\n")
+	for _, line := range linesout {
+		result := defaultRouteRE.FindStringSubmatch(line)
+		if result != nil {
```

Comment:
Perhaps this to ensure malformed/unexpected results don't panic the slice indexing on line 33.
```suggestion
		if len(result) > 4 {
```